src/services/iota.js

```diff
@@ -16,6 +16,19 @@ const IotaC = new IOTA({
   provider: IOTA_API.PROVIDER_C
 });
 
+const BUNDLE_SIZE = IOTA_API.BUNDLE_SIZE;
+
+let totalLength = 0;
+
+let indexes = {
```

Comment:
These state variables should be in their own reducer, but we can clean this up later so you can work on something more urgent.